spec/ruby/shared/types/rb_num2dbl_fails.rb

```diff
@@ -0,0 +1,17 @@
+#
+# Shared tests for rb_num2dbl related conversion failures.
+#
+# Usage example:
+#   it_behaves_like :rb_num2dbl_fails, nil, -> v { o = MyObject.new; o.blocking_io(param_to_convert: v) }
```

Comment:
**suggestion(nitpicking):** IMHO coercion to Float isn't related to blocking method call...

```suggestion
#   it_behaves_like :rb_num2dbl_fails, nil, -> v { o = A.new; o.foo(v) }
```

Reply:
Makes sense.